Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -168,6 +169,7 @@ final class GoogleSheetsManager: ObservableObject {
   }
   
   private func createSheetsValueRangeFrom(amount: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/67edf3325570da301c03450fdbbe6d8b96d1b452/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L171"></span>Function should have 5 parameters or less: it currently has 7<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:171</code></td>
    </tr>
  </tbody>
</table>